### PR TITLE
Update CMakeLists.txt

### PR DESCRIPTION
**Pull request by wm-0007**
Remove optional parameters from project() command on line 17
cmake version 3.5.1

original code:
project(celestia VERSION 1.7.0 DESCRIPTION "3D Universe simulator")

edited code:
project(celestia}

executed cmake command:
cmake .. -DENABLE_GTK=ON -DCMAKE_INSTALL_PREFIX=/home/user/Celestia-1.6.x/

error message:
CMake Error at CMakeLists.txt:17 (project):
  project with VERSION must use LANGUAGES before language names.

-- Configuring incomplete, errors occurred!

### CODE REVIEW

**Comment by eyvallah** [comment]
Or maybe we can set LANGUAGES property before this command.

**Comment by wm-0007** [comment]
The CMakeLists.txt lists on line 1: cmake_minimum_required(VERSION 3.1.0)

The project() command in cmake V3.1 does not have DESCRIPTION, only VERSION and LANGUAGES.

Editting the file to have line 17 read:
project(celestia VERSION 1.7.0 LANGUAGES CXX)
has no problem.

**Comment by 375gnu** [comment]
yeah, just checked `DESCRIPTION` was added in 3.9
Could you update your PR to
```
project(celestia VERSION 1.7.0 LANGUAGES CXX)
```

File: CMakeLists.txt [hunk 1]
```diff
@@ -14,7 +14,8 @@ if(POLICY CMP0072)
   endif()
 endif()
 
-project(celestia VERSION 1.7.0 DESCRIPTION "3D Universe simulator")
+project(celestia VERSION 1.7.0 LANGUAGES CXX)
```
Comment (375gnu) [inline CMakeLists.txt:17]: looks like `C` should be added as well

**Comment by 375gnu** [comment]
superseded by #719

**Pull request closed without merging**